security/src/main/java/io/airlift/security/AuthScheme.java

```diff
@@ -0,0 +1,31 @@
+package io.airlift.security;
+
+public enum AuthScheme
```

Comment:
As per our discussion, we keep AuthScheme in our internal usage. From the CLI, we expose only a flag.